api/notd/model.py

```diff
@@ -444,6 +444,7 @@ class GalleryToken:
     tokenMetadata: TokenMetadata
     tokenCustomization: Optional[TokenCustomization]
     tokenListing: Optional[TokenListing]
+    quantity: Optional[int]
```

Comment:
this shouldnt really be optional here should it? I think if we added to CollectionToken I can understand optional but here it feels like it should be mandatory